app/build.gradle.kts

```
@@ -48,6 +48,7 @@ dependencies {
     implementation("com.google.android.material:material:1.7.0")
     implementation("androidx.constraintlayout:constraintlayout:2.1.4")
     implementation("androidx.fragment:fragment-ktx:1.5.5")
+    implementation(project(mapOf("path" to ":domain")))
```

```suggestion
    implementation(project(":domain"))
```

Nit: 훨신 간략하게 모듈간 의존성 추가를 진행할 수 있습니다

---

IDEA에서 저렇게 간결하게 처리해줄 법한데.
수정하였습니다!